README.md

```diff
@@ -1,27 +1,84 @@
+[![Published on NPM](https://img.shields.io/npm/v/@polymer/iron-validator-behavior.svg)](https://www.npmjs.com/package/@polymer/iron-validator-behavior)
+[![Build status](https://travis-ci.org/PolymerElements/iron-validator-behavior.svg?branch=master)](https://travis-ci.org/PolymerElements/iron-validator-behavior)
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://webcomponents.org/element/@polymer/iron-validator-behavior)
 
-<!---
+## &lt;iron-validator-behavior&gt;
+Use `IronValidatorBehavior` to implement a custom input/form validator. Element
+instances implementing this behavior will be registered for use in elements that implement
+`IronValidatableBehavior`.
 
-This README is automatically generated from the comments in these files:
-iron-validator-behavior.html
+See: [Documentation](https://www.webcomponents.org/element/@polymer/iron-validator-behavior),
+  [Demo](https://www.webcomponents.org/element/@polymer/iron-validator-behavior/demo/demo/index.html).
 
-Edit those files, and our readme bot will duplicate them over here!
-Edit this file, and the bot will squash your changes :)
+## Usage
 
-The bot does some handling of markdown. Please file a bug if it does the wrong
-thing! https://github.com/PolymerLabs/tedium/issues
+### Installation
+```
+npm install --save @polymer/iron-validator-behavior
+```
 
--->
+### In a Polymer 3 element that is a validator
+```js
+import {PolymerElement, html} from '@polymer/polymer';
+import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
+import {IronValidatorBehavior} from '@polymer/iron-validator-behavior/iron-validator-behavior.js';
 
-[![Build status](https://travis-ci.org/PolymerElements/iron-validator-behavior.svg?branch=master)](https://travis-ci.org/PolymerElements/iron-validator-behavior)
+class SampleValidator mixinBehaviors([IronValidatorBehavior], PolymerElement){
+  static get template() {
+    return html``;
+  }
```

Comment:
Is the `template` getter necessary if this element just returns an empty template?

Reply:
i don't know! ill delete it